### PR TITLE
Commit_Branch_AAA

### PR DESCRIPTION
**Pull request by Dhanush2871996**
- added practice package under src/test/java/starter
- and added a class FirstTest.java under practice package

### CODE REVIEW

File: src/test/java/starter/practice/FirstTest.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package starter.practice;
```
Comment (AlphaWolf360) [inline src/test/java/starter/practice/FirstTest.java:1]: need changes here

File: src/test/java/starter/practice/FirstTest.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+public class FirstTest {
```
Comment (AlphaWolf360) [inline src/test/java/starter/practice/FirstTest.java:3]: also here

Comment (Dhanush2871996) [reply src/test/java/starter/practice/FirstTest.java:3]: got it

File: src/test/java/starter/practice/FirstTest.java (hunk 1 above)
Comment (Dhanush2871996) [reply src/test/java/starter/practice/FirstTest.java:1]: will be done

**Pull request merged**